manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+PackageLocale: en-US
+Publisher: HiBitSoftware
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: HiBit Startup Manager version 2.3.20.100
+# PackageUrl: 
+License: Freeware
+# LicenseUrl: 
+Copyright: Copyright @ 2017-2022 HiBitSoft.IR All Rights Reserved
+# CopyrightUrl: 
+ShortDescription: HiBit Startup Manager is a powerful Startup utility that Manage autostart programs by adding or removing apps, editing their properties and also Manage process, services, task scheduler and conrext menu.
+# Description: 
```

```suggestion
ShortDescription: Manage the List of All Applications that are Loaded Automatically When Windows Starts Up.
Description: HiBit Startup Manager is a powerful Startup utility that Manage autostart programs by adding or removing apps, editing their properties and also Manage process, services, task scheduler and context menu.
```
Short description from the website helps correlation